README.md

```diff
@@ -93,13 +93,15 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 ```javascript
 
-import { WechatyBuilder } from 'wechaty'
+import { WechatyBuilder } from "wechaty";
```

Comment:
Please remove the trailing `;` because it's not the style of the Wechaty community.

Reply:
Fixed